paf-mvp-demo-express/src/views/advertiser/index.hbs

```diff
@@ -2,853 +2,858 @@
 <html lang="html">
```

Comment:
If I understand well, the file is impacted by tabulation changes and some script tweaks.
Can we postpone unnecessary changes (like tabulation) so that we focus on what mater?

Tabulation or other cosmetic can have their own PRs.

Reply:
Yes, sorry. Fixed in new commit